html/_sources/files/overview.rst.txt

```diff
@@ -5,13 +5,12 @@ Overview
 
 ROBOTONT’s depth camera gives the ability to see like humans and map the entire environment in 3D. 
 With its omnidirectional wheels, the platform is free to move in any direction at any time. 
-Its transparent design, extendable hardware, and open source software offer plenty of room for customized extensions. 
+Its transparent design, extendable hardware, and open source software offer plenty of room for customized extensions. ROBOTONT currently includes the following out-of-the-box demos: 
+2D SLAM, 3D mapping, android and joystick based teleoperation, AR marker tracking and leap-motion controller based interaction.
 ROBOTONT is based on the most widely used software in robotics development called ROS (Robot Operating System). 
```

Comment:
Tekita siia kaks lõiku, mis on umbes sellised:
"ROBOTONT is an mobile ground robot with omnidirectional wheels to enable rapid movement in any direction at a desired heading. The sensor system includes a depth camera which allows it to see like humans and map the entire environment in 3D. The powerful on-board computer facilitates running high performance algorithms and libraries. The software stack is open-source and based on ROS (Robot Operating System). Its literally transparent design, modular hardware, and open-source software offer endeless custiomization and extendibility options.

ROBOTONT currently includes the following out-of-the-box demos:
- Android device, gamepad and keyboard based teleoperation,
- 2D mapping and SLAM,
- 3D mapping,
- AR marker tracking, and
- gesture-based human-robot interaction."